huxley/api/tests/auto.py

```diff
@@ -0,0 +1,69 @@
+# Copyright (c) 2011-2016 Berkeley Model United Nations. All rights reserved.
+# Use of this source code is governed by a BSD License (see LICENSE).
+
+from django.db import models
+from rest_framework import serializers
+
+from huxley.api.tests import RetrieveAPITestCase
+
+
+class RetrieveAPIAutoTestCase(RetrieveAPITestCase):
```

Comment:
I hope so! I figure we'll take it one step at a time, and the requirements of the other AutoTestCases will shape what the abstract one will look like.